dev/CommonStyles/TextBlock_themeresources.xaml

```diff
@@ -0,0 +1,59 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <x:Double x:Key="CaptionTextBlockFontSize">12</x:Double>
+    <x:Double x:Key="BodyTextBlockFontSize">14</x:Double>
+    <x:Double x:Key="BodyStrongTextBlockFontSize">14</x:Double>
```

Comment:
Fixed that in the new styles project, left the old styles project as is since we don't want to interfere with existing styles and as such need to either introduce a lightweight styling resource for the BodyStrongTextBlock fontsize or leave them out of that completely.